magento/pwa-studio
# Adjust page-level max-width

## Description

Update the max-width on a few page-level elements: header, main page, footer. Also fix some header spacing while we're in there.

## Related Issue

PWA-734

## Acceptance 

### Verification Stakeholders
@dpatil-magento 
@soumya-ashok 

### Specification

### Verification Steps
1. Visit the home page in a window with a viewport wider than `1440px`.
2. Verify the header content maxes out at `1440px`, while its border stretches edge-to-edge.
3. Verify the footer content maxes out at `1440px`, along with its border.
4. Verify no change at lower viewport widths.
5. Verify `/venia-new-home` still looks good.
6. Verify a product page still looks good.

## Screenshots / Screen Captures (if appropriate)

<img width="1723" alt="desktop" src="https://user-images.githubusercontent.com/2110986/88226492-ff273200-cc20-11ea-84a2-fe8009b43862.png">

<img width="612" alt="mobile" src="https://user-images.githubusercontent.com/2110986/88226504-04847c80-cc21-11ea-9960-eda852715b5b.png">


## Checklist
* I have updated the documentation accordingly, if necessary.
